contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -209,14 +217,15 @@ contract UniswapAnchoredView is UniswapConfig {
 
         // Calculate uniswap time-weighted average price
         FixedPoint.uq112x112 memory priceAverage = FixedPoint.uq112x112(uint224((nowCumulativePrice - oldCumulativePrice) / timeElapsed));
-        uint anchorPriceUnscaled = mul(priceAverage.decode112with18(), conversionFactor);
+        uint rawUniswapPriceMantissa = mul(priceAverage.decode112with18(), conversionFactor);
```

Comment:
Just to verify, `mantissa` always means `floor(num * 1e18)` encoding?

Reply:
yes.. why floor? when does `num * 1e18` cause rounding?

Reply:
I think mantissa means the raw value for an Exponential, which does have 18 decimals precision but probably shouldn't be confused with an amount of wei. Prices are ratios between quantities, and interpreted as Exponentials by the protocol, so mantissa makes sense